iot/api/CloudIotMqttExample/CloudIotMqttExample.csproj

```diff
@@ -1,19 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
I'd normally consider going from Core to Framework a retrograde step. Is this because one of the dependencies doesn't support Core?

Reply:
(Feel free to update the NodaTime dependency to 2.4.7 at the same time...)

Reply:
It's Bouncy Castle and one other. I did tried to update them but still, they don't support Core. Bouncy Castle will starting from 1.9.* bcgit/bc-csharp/pull/68.

I will update NodaTime and push again after Kokoro has finished.